packages/components/src/BrandMoment/_docs/BrandMoment.stories.tsx

```diff
@@ -1,102 +1,132 @@
 /* eslint-disable jsx-a11y/anchor-is-valid */
 import React from "react"
-import { Meta, StoryFn } from "@storybook/react"
-import { BrandMoment } from "@kaizen/brand-moment"
+import { Meta, StoryObj } from "@storybook/react"
+import isChromatic from "chromatic"
 import {
   BrandMomentCaptureIntro,
   BrandMomentPositiveOutro,
   BrandMomentError,
+  AnimatedSceneProps,
 } from "@kaizen/draft-illustration"
 import { Paragraph } from "@kaizen/typography"
-import arrowLeftIcon from "../../../icons/arrow-left.icon.svg"
-import arrowRightIcon from "../../../icons/arrow-right.icon.svg"
-import mailIcon from "../../../icons/email.icon.svg"
-import feedbackClassifyIcon from "../../../icons/feedback-classify.icon.svg"
-import securityTipIcon from "../../../icons/security-tip.icon.svg"
+// import { ArrowLeftIcon } from "~icons/ArrowLeftIcon"
+import { ArrowRightIcon } from "~icons/ArrowRightIcon"
+import { EmailIcon } from "~icons/EmailIcon"
+// import feedbackClassifyIcon from "../../../icons/feedback-classify.icon.svg"
+// import securityTipIcon from "../../../icons/security-tip.icon.svg"
```

Comment:
These can be fixed in a separate PR after the icons have been added, or if the PR adding the icons is merged first, I'll fix it here.